### ISSUE TITLE
untrapped error using TRUE/FALSE in model code

### ISSUE THREAD

**Issue by paciorek** [nimble-dev/nimble#875]
```
 code = nimbleCode({
     a <- pnorm(-2,0,1, lower.tail=FALSE)
 })
 m = nimbleModel(code)
defining model...
Error in replaceConstantsRecurse(x, constEnv, constNames) : 
  Error, hit end
```

**Comment by paciorek** [comment]
The same error occurs with this, although here it is debatable if we want TRUE converted to 1 as a model parameter...

```
 code = nimbleCode({
      a ~ dnorm(TRUE, 1)
 })
m = nimbleModel(code)
defining model...
Error in replaceConstantsRecurse(x, constEnv, constNames) : 
  Error, hit end

**Comment by paciorek** [comment]
Fixed by PR #904
